examples/005_convenience/customize.py

```diff
@@ -6,82 +6,116 @@
 It briefly demonstrates methods that modify the optics of kafe2 output.
 """
 
-from kafe2 import XYContainer, Fit, Plot
+from kafe2 import XYContainer, XYFit, Plot
```

Comment:
Why did you change Fit to XYFit? Is there any reason? Fit is a wrapper which automatically uses the correct fit class.